README.md

```diff
@@ -13,6 +13,14 @@ def deps do
 end
 ```
 
+## Installing the Airports data
+
+Airports depends on [tz_world](https://github.com/kimlai/tz_world) for timezone data. To install airports with timezone data, run:
+
+```elixir
+mix airports.update
```

Comment:
I like the addition of timezones! For confirmation, is it only necessary to run this command after `priv/airports.csv` has been changed? Or is this something users of the package have to do at their Elixir project as well?

Reply:
Only when `priv/airports.csv` changed since the `airports_with_tz.csv` file is already included in the package